scripts/zones/Southern_San_dOria/npcs/Femitte.lua

```diff
@@ -13,30 +13,15 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    local distantLoyaltiesProgress = player:getCharVar('DistantLoyaltiesProgress')
-    local distantLoyalties = player:getQuestStatus(xi.questLog.SANDORIA, xi.quest.id.sandoria.DISTANT_LOYALTIES)
     local wildcatSandy = player:getCharVar('WildcatSandy')
 
     if
         player:getQuestStatus(xi.questLog.SANDORIA, xi.quest.id.sandoria.LURE_OF_THE_WILDCAT) == xi.questStatus.QUEST_ACCEPTED and
         not utils.mask.getBit(wildcatSandy, 3)
     then
         player:startEvent(807)
-    elseif
-        player:getFameLevel(xi.fameArea.SANDORIA) >= 4 and
-        distantLoyalties == 0
-    then
-        player:startEvent(663)
-    elseif distantLoyalties == 1 and distantLoyaltiesProgress == 1 then
-        player:startEvent(664)
-    elseif
-        distantLoyalties == 1 and
-        distantLoyaltiesProgress == 4 and
-        player:hasKeyItem(xi.ki.MYTHRIL_HEARTS)
-    then
-        player:startEvent(665)
     else
-        player:startEvent(661)
+        return
```

Comment:
Unsure if the return lines in the npc luas is needed. I was unsure but these should eventually get removed once wildcat gets converted to IF. Same with the Rouva.lua file on line 22.

Reply:
Shouldn't be.

Reply:
You can remove the `else return` as it will go to the DefaultActions if none of the requirements are met.

Reply:
For both -